frontend/src/container/SideNav/NavItem/NavItem.tsx

```diff
@@ -31,7 +31,7 @@ export default function NavItem({
 
 				{isBeta && (
 					<div className="nav-item-beta">
-						<Tag color="magenta">Beta</Tag>
+						<Tag color="geekblue">Beta</Tag>
```

Comment:
let's use our design token variables here. `Color.XYZ`

Reply:
How do I find the design token variables? @vikrantgupta25

Reply:
https://github.com/SigNoz/signoz-core-ui/blob/main/packages/tokens/src/Colors/Colors.ts

Reply:
I tried using custom color, but AntD tag component by default gives a background for custom color

`<Tag bordered={false} color={Color.TEXT_ROBIN_500}>Beta</Tag>
`
![custome](https://github.com/user-attachments/assets/f802b065-a169-4623-9ebc-4f853f834bcb)

I think let's just use `geekblue`

Reply:
just checked they support only their preset colors in the `color` prop hence won't work , approved ✅